docs/templates/shared/_head.html.j2

```diff
@@ -2,6 +2,11 @@
     <meta charset="utf-8">
     <title>{{ headline }}</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta property="og:title" content="{{ headline }}">
+    <meta property="og:description" content="jq is a lightweight and flexible command-line JSON processor">
+    <meta property="og:type" content="website">
+    <meta property="og:image" content="{{ root }}/icon.svg">
```

Comment:
We can't use SVG for og:image.
https://indieweb.org/The-Open-Graph-protocol#Does_not_support_SVG_images

Reply:
👍  Fixed

Reply:
In the paragraph before that one, it also says you can't use relative paths: https://indieweb.org/The-Open-Graph-protocol#Disallows_relative_URLs
I think you need to prefix `{{ url }}`

Reply:
Also fixed 👍  generated locally and it looks like this:
```html
    <meta property="og:title" content="jq">
    <meta property="og:description" content="jq is a lightweight and flexible command-line JSON processor">
    <meta property="og:type" content="website">
    <meta property="og:image" content="https://jqlang.org/icon.png">
    <meta property="og:url" content="https://jqlang.org/">
```